pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -1229,6 +1241,12 @@ var _ = Describe("v1beta1 inference service controller", func() {
 						},
 					},
 				},
+				ModelStatus: v1beta1.ModelStatus{
+					TransitionStatus: v1beta1.InProgress,
+					ModelRevisionStates: &v1beta1.ModelRevisionStates{
+						ActiveModelState: v1beta1.Pending,
```

Comment:
Can we add a test cases for the error state? We can mock it by creating a pod with storage initializer error state